mod/kalvidassign/renderer.php

```diff
@@ -781,21 +781,6 @@ public function display_submissions_table($cm, $groupfilter = 0, $filter = 'all'
                 // No groups, do nothing
                 break;
             case SEPARATEGROUPS:
-                // If separate groups, but displaying all users then we must display only users
-                // who are in the same group as the current user
```

Comment:
@omridevk 
according to the comment - we do have to do something with groups in this case:
"who are in the same group as the current user"

so I'm not sure that completely removing it is the right way.
note that you removed the "break" as well, thus making this option use the same code as the next one - is this intentional?

since I don't know the insides of moodle with regards to this better than you, I would recommend just testing all different scenarios around "who this assignment is for" to make sure it all behaves as expected.